### ISSUE TITLE
where can I get this file: best_smallsigma.pth and What is the purpose of this file?

### ISSUE THREAD

**Issue by leozjr** [zsm1211/PnP-CASSI#2]
```
    model.load_state_dict(
        torch.load(r'/home/dgl/zhengsiming/self_train/check_points/best_smallsigma.pth'))
    model.eval()
```
it is at line 345 in the dvp_linear_inv_cassi.py

**Comment by zsm1211** [comment]
Sorry, it is the 'deep_denoiser.pth'. i forgot to change the name.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "zsm1211/PnP-CASSI"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年8月20日(星期五) 上午9:50
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[zsm1211/PnP-CASSI] where can I get this file: best_smallsigma.pth and What is the purpose of this file? (#2)





     model.load_state_dict(         torch.load(r'/home/dgl/zhengsiming/self_train/check_points/best_smallsigma.pth'))     model.eval()  
it is at line 345 in the dvp_linear_inv_cassi.py
 
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by XenoExia** [comment]
Could you offer me some sensing matrix data and the mearsured data of gisc system in the paper?

**Comment by zsm1211** [comment]
Thank you for your being interested in our paper.&nbsp;
The simulation sensing matrix is just a random binary (0-1) mask, it can be generated by yourself.
As for the data of gisc,&nbsp; i need to look it up on my old computer. I'll send it to you as soon as i find it.


Siming Zheng


------------------&nbsp; ------------------


 
Could you offer me some sensing matrix data and the mearsured data of gisc system in the paper?
